docs/integrations/unity-catalog-spiceai.md

```diff
@@ -0,0 +1,167 @@
+# Unity Catalog Spice OSS Integration
+
+Spice OSS is a unified SQL query interface and portable runtime to locally materialize, accelerate, and query datasets across databases, data warehouses, and data lakes.
+
+Unity Catalog and Databricks Unity Catalog can be used with [Spice OSS](https://github.com/spiceai/spiceai) as [Catalog Connectors](https://docs.spiceai.org/components/catalogs) to make catalog tables available for query in Spice.
+
+Follow the guide below to connect to Databricks Unity Catalog. See the [Unity Catalog quickstart](https://github.com/spiceai/quickstarts/blob/trunk/catalogs/unity_catalog/README.md) to connect to UC directly.
```

Comment:
Probably prefer to lead with the "any UC" quickstart and then link out to the Databricks UC guide separately in SpiceAI docs since it's more specific

Reply:
Agreed!   @lukekim - could you make that quick change and we'll merge?  Please tag me when you're done as in addition to merging, would love to chat with you about presenting your solution at one of the communitymeeutps, eh?! Thanks!

Reply:
@dennyglee done, thanks for your patience!